darts/tests/models/forecasting/test_regression_ensemble_model.py

```diff
@@ -198,6 +198,15 @@ def test_train_predict_global_models_multivar_with_covariates(self):
         ensemble.fit(self.seq1, self.cov1)
         ensemble.predict(10, self.seq2, self.cov2)
 
+    def test_train_predict_local_models_with_future_covariates(self):
+        ensemble_models = self.get_local_models()
+
+        # append model which supports future covariates
+        ensemble_models.append(Prophet())
```

Comment:
There are discussion to make Prophet optional (imported from another library, installation is not necessarily straight-forward): it would probably to use a simpler model to test this feature such as `LinearRegressionModel` or `RandomForest`.

Reply:
Makse sense.. will modify it